src/sentry/api/urls.py

```diff
@@ -607,7 +607,7 @@
         name='sentry-api-0-organization-events'
     ),
     url(
-        r'^organizations/(?P<organization_slug>[^\/]+)/events/(?P<project_slug>[^\/]+):(?P<event_id>(?:\d+|[A-Fa-f0-9]{32}))/$',
+        r'^organizations/(?P<organization_slug>[^\/]+)/events/(?P<project_slug>[^\/]+)/(?P<event_id>(?:\d+|[A-Fa-f0-9]{32}))/$',
```

Comment:
Do you mind switching this back to `:`? `/` implies a subresource to me

Reply:
ok.